### PR TITLE
Some UI improvemets for home and search page

### PR DESCRIPTION
**Pull request by JayaKrishnaNamburu**

### CODE REVIEW

**Comment by guybedford** [comment]
Thanks for working on this! Is it ready to merge, or are you looking for initial review here?

**Comment by JayaKrishnaNamburu** [comment]
We can check and merge, but i would like to make little more changes over this weekend too. We can do after that 👍

File: src/example-browser.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <script type="module">
-  import { render, html, svg } from 'uhtml/async';
-  import jspm from 'https://api.github.com/orgs/jspm' assert { type: "json" };
+  import { render, html, svg } from "uhtml/async";
+  // import jspm from 'https://api.github.com/orgs/jspm' assert { type: "json" };
```
Comment (fusionstrings) [inline src/example-browser.html:3]: why are these commented?

**Review by fusionstrings — APPROVED** [review]
thanks for this PR. Sorry for late review

File: src/example-browser.html (hunk 1 above)
Comment (JayaKrishnaNamburu) [reply src/example-browser.html:3]: In dev mode. Every time we refresh, it's making a call to github. After certain calls, github started to block my requests 😬. So, I thought we could make it static and forgot to change it back

**Pull request merged**